### PR TITLE
add yaqd-attune

### PR DESCRIPTION
**Pull request by untzag**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ksunden** [comment]
Willing

**Comment by ksunden** [comment]
@conda-forge/staged-recipes

I believe this is ready for merging

**Review by carterbox — CHANGES_REQUESTED** [review]
It looks like this is one package, but the yaqd-attune-gui package just adds additional dependencies, yes? In that case, yaqd-attune-gui should be an empty meta-package which depends on yaqd-attune plus the addtional dependencies.

Right now, the two package have duplicate files which is why the build logs show an [overlap warning.](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=497414&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=3556)

[Here is an example](https://github.com/conda-forge/importlib_resources-feedstock/blob/main/recipe/meta.yaml) of a recipe with a main package plus an empty metapackage.

File: recipes/yaqd-attune/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  commands:
+    - pip check
+  requires:
+    - pip
+
+outputs:
+  - name: yaqd-attune
+    build:
+      noarch: python
+      script: python -m pip install . -vv
```
Comment (carterbox) [inline recipes/yaqd-attune/meta.yaml:27]: The reason that `{{ PYTHON }}` doesn't work here is that script in the output section expects the name of a script not a script literal.

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#specifying-files-to-include-in-output

**Pull request closed without merging**